sbi/inference/base.py

```diff
@@ -179,6 +187,66 @@ def get_simulations(
 
         return theta, x, prior_masks
 
+    def append_simulations(
+        self,
+        theta: Tensor,
+        x: Tensor,
+        exclude_invalid_x: bool = False,
+        from_round: int = 0,
+        algorithm: str = "SNLE or SNRE",
+        data_device: Optional[str] = None,
+    ) -> "NeuralInference":
+        r"""Store parameters and simulation outputs to use them for later training.
+
+        Data are stored as entries in lists for each type of variable (parameter/data).
+
+        Stores $\theta$, $x$, prior_masks (indicating if simulations are coming from the
+        prior or not) and an index indicating which round the batch of simulations came
+        from.
+
+        Args:
+            theta: Parameter sets.
+            x: Simulation outputs.
+            exclude_invalid_x: Whether invalid simulations are discarded during
+                training. If `False`, The inference algorithm raises an error when invalid simulations are
+                found. If `True`, invalid simulations are discarded and training
+                can proceed, but this gives systematically wrong results.
+            from_round: Which round the data stemmed from. Round 0 means from the prior.
+                With default settings, this is not used at all for the inference algorithm. Only when
+                the user later on requests `.train(discard_prior_samples=True)`, we
+                use these indices to find which training data stemmed from the prior.
+            data_device: Where to store the data, default is on the same device where
```

Comment:
add doctoring for `algorithm`. Note that the message works also for `APT`, so in principle one could pass `algorithm="APT"` or `"multi-round SNPE"` as well.

Reply:
Ah good point, thanks! The new docstring is now somewhat general. Let me know if you want explicit examples mentioned.